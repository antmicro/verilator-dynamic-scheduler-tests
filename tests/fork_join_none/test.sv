module t (/*AUTOARG*/
   // Inputs
   clk
   );
   input clk;

   initial begin
      fork
         begin
            #1;
            $write("forked process\n");
            $write("*-* All Finished *-*\n");
            $finish;
         end
      join_none
      $write("main process\n");
   end
endmodule
