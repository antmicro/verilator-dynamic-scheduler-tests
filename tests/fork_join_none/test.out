main process
forked process
*-* All Finished *-*
